File: Java/Day03Assignment/Employee.java
package com.sonata;

public class Employee {
	int empId;
	String empName;
	double empSal;

	public Employee(int id,String name,double sal) {
		this.empId = id;
		this.empName = name;
		this.empSal = sal;
	}
	public void display() {
		System.out.println("Employee ID :" + empId);
		System.out.println("Employee Name :" + empName);
		System.out.println("Employee Salary :" + empSal);
	}
	public double salCal(double HRA,double DA) {
		double total = this.empSal + HRA + DA;
		return total;
	}
	public static void main(String[] args) {
		
	}

}

File: Typescripts/Employee.js
"use strict";
exports.__esModule = true;
exports.Employee = void 0;
var Employee = /** @class */ (function () {
    function Employee(i, n, s) {
        this.empID = 123;
        this.empName = "Rithesh";
        this.empSal = 2345;
        this.empID = i;
        this.empName = n;
        this.empSal = s;
    }
    Employee.prototype.display = function () {
        console.log(this.empID);
        console.log(this.empName);
        console.log(this.empSal);
    };
    return Employee;
}());
exports.Employee = Employee;
var e1 = new Employee(12, "abc", 3455);
e1.display();
